Core/GDCore/Extensions/Builtin/BaseObjectExtension.cpp

```diff
@@ -828,8 +828,8 @@ void GD_CORE_API BuiltinExtensionsImplementer::ImplementsBaseObjectExtension(
       .AddParameter("string", _("Timer's name"));
 
   obj.AddAction("PauseObjectTimer",
-                _("Pause a timer"),
-                _("Pause a timer."),
+                _("Pause an object timer"),
+                _("Pause an object timer."),
```

Comment:
We should also do that for UnPauseObjectTimer, RemoveObjectTimer and the ObjectTimerElapsedTime expression ;)